src/App.tsx

```diff
@@ -146,10 +121,10 @@ export const App: React.FC = () => {
 
         {/* show only one message at a time */}
         Unable to add a todo
-        <br />
+        {/* <br />
         Unable to delete a todo
         <br />
-        Unable to update a todo
+        Unable to update a todo */}
```

Comment:
just delete it instead of commenting

The error shouldn't be always visible

Implement it as it's required:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/86923089/bbac4edf-de16-4410-a78a-954f09e944e7)


Feel free to ask in the chat if you need help